contracts/Lockbox.cdc

```diff
@@ -0,0 +1,468 @@
+/*
+
+    Lockbox implements the functionality required to manage FLOW
+    buyers locked tokens from the token sale.
+
+    Each token holer gets two accounts. One account is their locked token
+    account. It will be jointly controlled by the user and Dapper Labs.
+    No actions will be able to be performed with this account without authorization
+    from Dapper Labs. Locked tokens are stored in this account and Dapper Labs
+    with authorize withdrawals every time a milestone is passed in the
+    vesting period.
```

Comment:
Similar to the above, I think we should be very clear about what the token admin can do. Something like this maybe?

```
The token administrator must co-sign the transfer of any locked tokens. The token admin cannot interact with the account without approval from the token holder, except to deposit additional locked FLOW or to unlock existing FLOW at each milestone in the token vesting period.